arch/AArch64/AArch64Mapping.c

```diff
@@ -2534,8 +2559,8 @@ void AArch64_set_detail_op_imm_range(MCInst *MI, unsigned OpNum,
 		if (AArch64_get_detail_op(MI, 0)->type == AARCH64_OP_SME) {
 			AArch64_set_detail_op_sme(MI, OpNum,
 						  AARCH64_SME_MATRIX_SLICE_OFF_RANGE,
-						  AARCH64LAYOUT_INVALID, FirstImm,
-						  Offset);
+						  AARCH64LAYOUT_INVALID, (int) FirstImm,
+						  (int) Offset);
```

Comment:
Why here it cast to `int` while in the previous change the last argument was of the `int64_t` type? Also using `int` for offset looks suspicious

Reply:
I use `int8_t` here because I need to have an invalid value like -1. But yeah. I should go with `uint16_t` and invalid is `UINT16_MAX`